decidim-proposals/app/views/decidim/proposals/admin/proposals/_proposal-tr.html.erb

```diff
@@ -1,15 +1,49 @@
-<tr data-id="<%= proposal.id %>" <%= "data-published-state=false" if proposal.answered? && !proposal.published_state? %>>
+<tr data-id="<%= proposal.id %>"
+    <%= "data-published-state=false" if proposal.answered? && !proposal.published_state? %>>
   <td>
-    <%= check_box_tag "proposal_ids_s[]", proposal.id, false, class: "js-check-all-proposal js-proposal-list-check  js-proposal-id-#{proposal.id}" %><br>
+    <%= check_box_tag "proposal_ids_s[]", proposal.id, false, class: "js-check-all-proposal js-proposal-list-check  js-proposal-id-#{proposal.id}" %>
+    <br>
   </td>
   <td>
     <%= proposal.id %><br>
   </td>
   <td>
-    <%= link_to(
-     decidim_html_escape(present(proposal).title).html_safe,
-     proposal_path(proposal)
-    ) %><br>
+    <div class="table__title-block">
+      <span class="table__title-icon">
+        <% if unanswered_proposals_overdue?(proposal) || evaluating_proposals_overdue?(proposal) %>
+          <%= icon "timer", class: "text-alert" %>
+        <% elsif grace_period_unanswered?(proposal) || grace_period_evaluating?(proposal) %>
+          <%= icon "datetime", class: "text-warning" %>
+        <% end %>
+      </span>
+      <div class="table__list-title">
+        <%= link_to(
+              decidim_html_escape(present(proposal).title).html_safe,
+              proposal_path(proposal)
+            ) %>
+        <% if unanswered_proposals_overdue?(proposal) %>
+          <div class="help-text-overdue text-alert">
+            <%= time_ago_in_words(last_day_to_answer(proposal)).gsub("ago", "") %>
+            overdue
```

Comment:
```suggestion
            <%= time_ago_in_words(last_day_to_answer(proposal), scope: "decidim.proposals.admin.answer_overdue") %>
```
We should use I18n instead of a simple text.
Also, using `gsub`here is unreliable because every language is different. In fact, originally, the "ago" word is not part of the method `time_ago_in_words` but it has been overriden in decidim (see `decidim-core/config/locales/en.yml`).

I think the prefered approach would be to use a custom scope and put there our translations in (`decidim-proposals/config/en.yml`), adding this content for instance:
```yaml
  decidim:
    proposals:
      admin:
        answer_overdue:
          datetime:
            distance_in_words:
              about_x_hours:
                one: about 1 hour overdue
                other: about %{count} hours overdue
              about_x_months:
                one: about 1 month overdue
                other: about %{count} months overdue
              half_a_minute: half a minute overdue
              less_than_x_minutes:
                one: less than a min. overdue
                other: less than %{count} min. overdue
              less_than_x_seconds:
                one: right now overdue
                other: less than %{count} sec. overdue
              x_days:
                one: 1 day ago overdue
                other: "%{count} days ago overdue"
              x_hours:
                one: 1 hour ago overdue
                other: "%{count} hours ago overdue"
              x_minutes:
                one: 1 min. ago overdue
                other: "%{count} min. ago overdue"
              x_seconds:
                one: 1 sec. ago overdue
                other: "%{count} sec. ago overdue"
                zero: right now overdue
        answer_pending:
          datetime:
            distance_in_words:
              about_x_hours:
                one: about 1 hour to answer
                other: about %{count} hours to answer
              about_x_months:
                one: about 1 month to answer
                other: about %{count} months to answer
              half_a_minute: half a minute to answer
              less_than_x_minutes:
                one: less than a min. to answer
                other: less than %{count} min. to answer
              less_than_x_seconds:
                one: right now to answer
                other: less than %{count} sec. to answer
              x_days:
                one: 1 day ago to answer
                other: "%{count} days ago to answer"
              x_hours:
                one: 1 hour ago to answer
                other: "%{count} hours ago to answer"
              x_minutes:
                one: 1 min. ago to answer
                other: "%{count} min. ago to answer"
              x_seconds:
                one: 1 sec. ago to answer
                other: "%{count} sec. ago to answer"
                zero: right now to answer
        evaluate_pending:
          datetime:
            distance_in_words:
              about_x_hours:
                one: about 1 hour to evaluate
                other: about %{count} hours to evaluate
              about_x_months:
                one: about 1 month to evaluate
                other: about %{count} months to evaluate
              half_a_minute: half a minute to evaluate
              less_than_x_minutes:
                one: less than a min. to evaluate
                other: less than %{count} min. to evaluate
              less_than_x_seconds:
                one: right now to evaluate
                other: less than %{count} sec. to evaluate
              x_days:
                one: 1 day ago to evaluate
                other: "%{count} days ago to evaluate"
              x_hours:
                one: 1 hour ago to evaluate
                other: "%{count} hours ago to evaluate"
              x_minutes:
                one: 1 min. ago to evaluate
                other: "%{count} min. ago to evaluate"
              x_seconds:
                one: 1 sec. ago to evaluate
                other: "%{count} sec. ago to evaluate"
                zero: right now to evaluate
```

Reply:
see https://apidock.com/rails/v5.2.3/ActionView/Helpers/DateHelper/distance_of_time_in_words for more info about this method